config/runtime.exs

```diff
@@ -5,6 +5,16 @@ import Config
 # system starts, so it is typically used to load production configuration
 # and secrets from environment variables or elsewhere. Do not define
 # any compile-time configuration in here, as it won't be applied.
+
+config :cors_plug,
+  max_age: 86400,
+  methods: ["GET"],
+  headers: ["Authorization", "Content-Type"],
+  origin: System.get_env("CLIENT_ORIGIN_URL") ||
```

Comment:
@manusajith maybe we should use `fetch_env` here?

Reply:
@sestrella I used `fetch_env` here to customise the error message. The guide had a specific error message for this case.

Reply:
@manusajith got it, in that case, I think we should be more specific about what ENV variable is missing:

```
The required environment variable CLIENT_ORIGIN_URL is missing. Check .env file.
```